README.md

```diff
@@ -1 +1,23 @@
 # Server
+
+## Version Control Guidelines
+
+### Commits Convention
+
+The [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/)
+should be followed using the following types:
+
+| Type       | Related Changes                                                  |
+| :--------- | :--------------------------------------------------------------- |
+| `style`    | Restyling widgets or Updating theme, without affecting any logic |
```

Comment:
We should remove this. it doesn't make sense here. We can use `test` instead for _adding or updating unit tests_.